packages/node_modules/@ciscospark/plugin-feature/src/feature.js

```diff
@@ -81,7 +84,12 @@ const Feature = SparkPlugin.extend({
       resource: `features/users/${this.spark.device.userId}/toggles`,
       body: featureList
     })
-      .then((res) => this.spark.device.features[`user`].add(res.body, {merge: true}));
+      .then((res) => {
+        const partitionedToggles = partition(res.body.featureToggles, {type: `USER`});
+        this.spark.device.features.user.add(partitionedToggles[0], {merge: true});
+        this.spark.device.features.developer.add(partitionedToggles[1], {merge: true});
+        return this.spark.device.features;
```

Comment:
I'm not sure we want to return the features object here. Ideally, developers are only accessing feature values via `spark.features.getFeature`. If we encourage raw access, I could imagine things might get changed inappropriately. Is this return value getting used somewhere?